HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 History
 =======
 
+0.6.0 (2023-08-28)
```

Comment:
TODO: fix that date before merging -- thought I'd get it out yesterday but got distracted, whoops 🐠

Reply:
```suggestion
0.6.0 (2023-08-30)
```